src/legacy/core_plugins/kibana/public/visualize/embeddable/visualize_embeddable_factory_provider.ts

```diff
@@ -17,13 +17,15 @@
  * under the License.
  */
 
-import { VisualizeEmbeddableFactory } from './visualize_embeddable_factory';
 import { EmbeddableFactoriesRegistryProvider } from 'ui/embeddable/embeddable_factories_registry';
+import { KibanaConfig, SavedVisualizations } from '../types';
+import { VisualizeEmbeddableFactory } from './visualize_embeddable_factory';
 
-export function visualizeEmbeddableFactoryProvider(Private) {
+export function visualizeEmbeddableFactoryProvider(Private: any) {
```

Comment:
**Question**: Is there a difference between `Private` here and the `IPrivate` type defined in `src/ui/public/private/private.d.ts`? I'm not super familiar, but thought I'd point it out in case it can be reused.

https://github.com/elastic/kibana/blob/91f50afead6547fcc266995bef417843abaf5235/src/ui/public/private/private.d.ts#L20